app/views/rivalries/new.slim

```diff
@@ -1,6 +1,10 @@
 - title 'Set a rival'
```

Comment:
This page has some formatting issues.  Most notably is no padding on the avatar, and the same follow issue down below.

![chrome_2018-06-08_19-37-36](https://user-images.githubusercontent.com/7030913/41186880-81a1cf82-6b53-11e8-89b6-6da6dd8b87df.png)

Reply:
Good catch, redid it all to look decent. Probably still want to revisit at some point but it's back to looking good enough for now.